scripts/versions.json

```diff
@@ -1,4 +1,32 @@
 {
+  "1.0.1": {
```

Comment:
Did you do the automatic version numbering for BoM, i was expecting to have a 1.1.0 since we're adding a new package 🤔

Reply:
Okay, just checking :)